src/libs/AppStateMonitor/index.js

```diff
@@ -26,6 +26,9 @@ function addBecameActiveListener(callback) {
     }
     const appStateChangeSubscription = AppState.addEventListener('change', appStateChangeCallback);
     return () => {
+        if (!appStateChangeSubscription) {
+            return;
+        }
         appStateChangeSubscription.remove();
```

Comment:
I think this code change still makes sense regardless right? It might be redundant with the RN upgrade if that makes it so `appStateChangeSubscription` is never null, but it wouldn't hurt to have this check.

Reply:
Agreed.

Reply:
Dope then I think it's safe to merge!

Reply:
FWIW, I think including the null check is a good idea even in RN 0.65+, because, while unlikely, I think we could potentially have a race condition like this if a component mounts and unmounts almost instantly:

1. Constructor initializes `appStateChangeSubscription` to `null`.
1. Component mounts, `componentDidMount` runs and starts creating the subscription. 
1. Component unmounts and `componentWillUnmount` runs before `componentDidMount` finishes creating the subscription. So the subscription is still null 💥

Reply:
Noted. Thanks for the explanation :)